lib/src/deprecation.dart

```diff
@@ -18,167 +18,118 @@ enum Deprecation {
   // Checksum: c57ab2eb07ab1df48581b8484ef9bdbad0ddceaa
 
   /// Deprecation for passing a string directly to meta.call().
-  callString(
-    'call-string',
-    deprecatedIn: '0.0.0',
-    description: 'Passing a string directly to meta.call().',
-  ),
+  callString('call-string',
```

Comment:
changes in this file are the result of running `dart run grinder`

Reply:
ok so it wasn't just me then!